tests/Feature/Traits/CatchFailures.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Tests\Feature\Traits;
+
+use Illuminate\Testing\TestResponse;
+
+trait CatchFailures
+{
+	protected function assertStatus(TestResponse $response, int $expectedStatusCode): void
+	{
+		if ($response->getStatusCode() === 500) {
+			$exception = $response->json();
+			$this->trimException($exception);
+			// $exception['trace'] = array_slice($exception['trace'], 0 ,3);
```

Comment:
As always: I don't like commented code without a comment why it has been commented, whether it will be required again eventually, under which conditions it may be required again, what needs to be done to get there, etc. pp.

If the code is just out-dated, delete it.

Reply:
removed.